src/index.html

```diff
@@ -35,11 +37,11 @@ <h1>PING PONG</h1>
           </div>
           <div class="row">
             <p>BEST OF</p>
-            <input type="radio" id="1" name="bestof" value="1" checked="checked">
+            <input id="radios" type="radio" id="1" name="bestof" value="1" checked="checked">
             <label for="1">1</label>
-            <input type="radio" id="3" name="bestof" value="3">
+            <input id="radios" type="radio" id="3" name="bestof" value="3">
             <label for="3">3</label>
-            <input type="radio" id="5" name="bestof" value="5">
+            <input id="radios" type="radio" id="5" name="bestof" value="5">
             <label for="5">5</label>
```

Comment:
Why is `id="radios"` required here? Can it be removed?

Reply:
yes, I tried to make changes but forgot to remove it.